kedro/io/cached_dataset.py

```diff
@@ -93,10 +93,14 @@ def _from_config(config: dict, version: Version | None) -> AbstractDataset:
         return AbstractDataset.from_config("_cached", config)
 
     def _describe(self) -> dict[str, Any]:
-        return {
-            "dataset": self._dataset._describe(),
-            "cache": self._cache._describe(),
+        return {"dataset": self._dataset._describe(), "cache": self._cache._describe()}
```

Comment:
Thank you for the change! It's much better now to keep `describe` decoupled from the pretty print